rest-api-spec/src/main/resources/rest-api-spec/api/synonym_rule.put.json

```diff
@@ -0,0 +1,43 @@
+{
+  "synonym_rule.put": {
+    "documentation": {
+      "url": "https://www.elastic.co/guide/en/elasticsearch/reference/master/put-synonym-rule.html",
+      "description": "Creates or updates a synonym rule for a synonym set"
```

Comment:
```suggestion
      "description": "Creates or updates a synonym rule inside a synonym set"
```

Reply:
"in" would probably be better, right? (Ah, English prepositions...) 😞

Reply:
In would be good!